e2e/testdata/live-apply/json-output/config.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+parallel: true
+kptArgs:
+  - "--output=json"
+stdOut: |
```

Comment:
ignore my previous comment. It looks like you are adding an explicit test for testing json-output.

Reply:
Yeah, having the same test looking at multiple output formats turned out to be difficult to manage. So removed that functionality and instead have separate tests for alternative output formats.